src/js/common/schemaform/FormPage.jsx

```diff
@@ -140,7 +140,7 @@ class FormPage extends React.Component {
           pagePerItemIndex={params ? params.index : undefined}
           uploadFile={this.props.uploadFile}
           prefilled={this.props.form.prefillStatus === PREFILL_STATUSES.success}
-          onChange={this.onChange}
+          onChange={_.once(this.onChange)} // Wrapped in _.once to prevent message events from retriggering validation
```

Comment:
I still see validations happening twice. I think the issue is more likely in how RJSF handles state (i.e. it has local state and we also have Redux state).

Reply:
Thanks @jbalboni, `validate` is now wrapped in `once`, and only fires once per change.